### PR TITLE
Executable migration script

### PR DESCRIPTION
**Pull request by juliangoacher**
This PR registers the `trail-core` migration script as an npm binary, the script can now be run using `npx migrate` after the trail package is installed.

Notes:
* The script requires a configuration file with appropriate settings before it can run;
* The project README says that the migration script can be passed a version argument via the command line, but this doesn't seem to be the case; the README should probably be updated and describe the configuration options instead.

### CODE REVIEW

**Comment by mcollina** [comment]
Currently blocked on #24

**Comment by thomheymann** [comment]
I've tested this locally and binary scripts from `trail-core` install and work correctly when installing e.g. `trail-fastify-plugin`:

```
➜  test-project npm i @nearform/trail-fastify-plugin --registry http://localhost:4873

+ @nearform/trail-fastify-plugin@3.0.1
added 87 packages from 62 contributors and audited 128 packages in 11.848s

found 0 vulnerabilities

➜  test-project npx trail-database-init
✔ Database trails created successfully!
```

File: packages/trail-core/README.md [hunk 1]
```diff
@@ -8,19 +8,30 @@ trail-core is the core package. It is responsible for manipulating audit trails
...
 To install via npm:
 
-```
-npm install @nearform/trail-core
-```
+    npm install @nearform/trail-core
+
+Trail requires an instance of Postgres (version 9.5+) to function correctly. For simplicity, a preconfigured `docker-compose` file has been provided:
+
+    docker-compose up
```
Comment (mcollina) [inline packages/trail-core/README.md]: where is the docker-compose file live? They should download it from somewhere, can you put a full URL?

Comment (thomheymann) [reply packages/trail-core/README.md]: It's already bundled with the module, I'll add instructions

**Pull request merged**